resources/styles/mixins.less

```diff
@@ -80,6 +82,14 @@
     a {
       font-weight: 400;
       color: @tutor-neutral !important;
+      span {
```

Comment:
Could you add a class to this element so it is not selecting `span`? Maybe `react-bootstrap` does not have an easy way to do this though